### DISCUSSION TITLE
What's the process of upgrading an existing Azure Redcap deployment

### DISCUSSION THREAD

**Discussion by SvDellen** [microsoft/azure-redcap-paas#56] [Q&A] [answered]
Hi,

How can I upgrade an existing Redcap Azure deployment to a higher version. In other words, what steps to take to install the latest version.

Thanks in advance

Sijtze van Dellen

**Comment by SvenAelterman** [comment, marked as answer]
We are working on a reference model to enable updates to REDCap to take place. Most likely, this will involve deploying a new App Service with the new build of REDCap.

In the longer term, there might be options to collaborate with the developers to make Easy Upgrade succeed on Azure App Service.
